include/zephyr/drivers/i2s.h

```diff
@@ -142,6 +142,50 @@ typedef uint8_t i2s_fmt_t;
  */
 #define I2S_FMT_DATA_FORMAT_RIGHT_JUSTIFIED (4 << I2S_FMT_DATA_FORMAT_SHIFT)
 
+/**
+ * @brief Time Division Multiplexed (TDM) mode A Data Format.
+ *
+ * Serial data is transmitted with the MSB first. Both Frame Synchronization
+ * (FSYNC) and Serial Data (SD) signals are sampled on the rising edge of the
+ * Bit Clock (BCLK) signal. The bits within the data word are left justified.
+ * The frame sync is one clock cycle long.
+ *
+ * Mode A: the beginning of the channel data is delayed one period of BCLK
+ * following the rising edge of the FSYNC signal.
+ *
+ *          .-. .-. .-. .-. .-. .-. .-. .-. .-. .-. .-. .-. .-. .-. .-. .-. .-.
+ *    BCLK -' '-' '-' '-' '-' '-' '-' '-' '-' '-' '-' '-' '-' '-' '-' '-' '-' '-
+ *            .---.                                                           .-
+ *   FSYNC ---'   '-----------------------------------------------------------'
```

Comment:
It seems that FSYNC and WS are interchangeable terms for the same signal- should we use `WS` in these diagrams since that is what Zephyr uses for the remainder of diagrams in this documentation?

Reply:
Done